openapi.yaml

```diff
@@ -3911,9 +3951,10 @@ paths:
     get:
       tags:
         - Callbacks
-      summary: List callback instances
-      description: Get a list of callbacks instances
-      operationId: getCallbackInstances
+      x-sdk-operation: listCalls
+      summary: List callback calls
+      description: List of calls to the callback
+      operationId: getCallbackCalls
```

Comment:
"Callback instances" are confusing - it should really be "Callback calls", the calls made to the callback endpoint.

I've updated this and all other related endpoints to reflect this.